docker/pythonpath_dev/superset_config.py

```diff
@@ -106,6 +106,10 @@ class CeleryConfig(object):
 WEBDRIVER_BASEURL = "http://superset:8088/"
 # The base URL for the email report hyperlinks.
 WEBDRIVER_BASEURL_USER_FRIENDLY = WEBDRIVER_BASEURL
+# customize the polling time of each engine. The default time is 5 seconds
+DB_POLL_INTERVAL_SECONDS = {
+    "hive": int(timedelta(seconds=5).total_seconds()),
+}
```

Comment:
I'm not sure if this is needed, as it's not changing the defaults.

Reply:
I can restore the previous code without changing this file